cmake/OpenCVFindLAPACK.cmake

```diff
@@ -156,8 +163,16 @@ if(WITH_LAPACK)
     if(WIN32 AND NOT OPENCV_LAPACK_SHARED_LIBS)
       set(BLA_STATIC 1)
     endif()
-    find_package(LAPACK)
+    find_package(LAPACK CONFIG)
```

Comment:
> CONFIG

Why do we need to block CMake's LAPACK module?

Reply:
I just use CONFIG so as to have access to LAPACK_VERSION. If I don't add CONFIG LAPACK_VERSION is undefined.